### DISCUSSION TITLE
Is it possible to raise the landscape?

### DISCUSSION THREAD

**Discussion by rrbailey89** [Stellarium/stellarium#2741] [Q&A]
I took a pano, and I do not think I had my camera at the correct height. This means that Stellarium shows objects as visible when trees or buildings should block them. Is it possible to elevate the landscape using some setting? Or should I retake the pano?

**Comment by gzotti** [comment]
You must configure the pano correctly as described in the User Guide. Probably some photo or pano editing software can help visually adjusting a non-so-perfect pano.

**Reply by rrbailey89** [reply]
Attached is a screenshot of what I have and what I am referring too. I used a smartphone to make a spherical panorama and edited it in photoshop. Polaris should be behind the tree. Does it look this way due to improper processing or because I was not holding the phone at the correct height? 

![image](https://user-images.githubusercontent.com/6540217/194783506-6aa8b4e1-e9ed-4191-bfab-f31bc1ae3e3d.png)

**Reply by gzotti** [reply]
I don't know how it _should_ look like... 
There is no adjustment magic built into Stellarium, other than a vertical rotation to adjust azimuth. The pano has to be created/straightened/adjusted with other means as described in the User Guide.
